build/ctags_test.go

```diff
@@ -44,8 +44,8 @@ func TestTagsToSections(t *testing.T) {
 }
 
 func TestTagsToSectionsMultiple(t *testing.T) {
-	c := []byte("class Foob { int x; int b; }")
-	// ----------012345678901234567890123456789
+	c := []byte("class Foo { int x; int b; }")
```

Comment:
The original test only works because we process `x` before `b`, which is hardcoded in the test. For SystemVerilog, for example, the symbols for the following snipped 

```
typedef enum { x, b } Foob
```
appear in the order `Foob`, `x`, `b` and the original code would drop `x` and `b`.

Reply:
we could check for word boundaries instead of just calling `bytes.Index`, which would be more expensive but cover both cases. WDYT?